lms/static/scripts/frontend_apps/components/BasicLtiLaunchApp.js

```diff
@@ -298,6 +301,19 @@ export default function BasicLtiLaunchApp({ rpcServer }) {
           <b>To fix this problem, try reloading the page.</b>
         </Dialog>
       )}
+      {errorState === 'error-unrecoverable' && (
+        <Dialog
+          title="Something went wrong"
+          contentClass="BasicLtiLaunchApp__dialog"
+          role="alertdialog"
+        >
+          <ErrorDisplay
+            message="There was an unrecoverable problem with the server"
+            error={error}
+          />
+          <b>To fix this problem, try reloading the page.</b>
```

Comment:
An unfortunate problem here is that we've lost information about the context of what the frontend was trying to do that failed.